### ISSUE TITLE
HKID ERROR 

### ISSUE THREAD

**Issue by s20041152** [Edditoria/validid#6]
can't check start with xg or n

**Comment by Edditoria** [comment]
Hi,

**For HKID starting with 'XG':**

Would you please check again? You can add some tests in `testData.coffee` file, e.g.

```
{id: 'XG9876549', cardType: 'hkid', expect: true}
{id: 'XG9876543', cardType: 'hkid', expect: false}
```

Put a real ID number if you have one. And then run `npm run test`.

---

**For HKID starting with 'N':**

Okay, I have seen the news about the wrong issues of ID number. I didn't expect someone can get an ID with suffix letter "N" in 2019.

Anyway, I will add this support in next version. Coming very soon! :)

Update on "N" issue: Solved in v1.2.0 (commit b57eea8c )

---

If you want me to test any real ID number, welcome to telegram me: <https://telegram.me/edditoria>.

Please don't post any real ID number (to protect privacy :)

**Comment by s20041152** [comment]
Thx I will try it.
